### PR TITLE
Add username and password options

### PR DESCRIPTION
**Pull request by zschwarz**

### CODE REVIEW

File: cli-paho-java/src/main/java/com/redhat/mqe/amc/Client.java [hunk 1]
```diff
@@ -121,6 +124,10 @@ OptionParser populateOptionParser(OptionParser parser) {
 
         willDestination = parser.accepts("conn-will-destination", "will topic name").withRequiredArg().ofType(String.class);
 
+        username = parser.accepts("conn-username", "username").withRequiredArg().ofType(String.class).defaultsTo("");
+
```
Comment (jdanekrh) [inline cli-paho-java/src/main/java/com/redhat/mqe/amc/Client.java]: I'd remove the empty line here

File: cli-paho-java/src/main/java/com/redhat/mqe/amc/Client.java [hunk 2]
```diff
@@ -175,24 +187,35 @@ void printMessage(String topic, MqttMessage message) {
         }
     }
 
-    static MqttConnectOptions setConnectionOptions(MqttConnectOptions connectOptions) {
-        connectOptions.setUserName("admin");
-        connectOptions.setPassword("admin".toCharArray());
+    static MqttConnectOptions setConnectionOptions(MqttConnectOptions connectOptions, String username, String password) {
+
```
Comment (jdanekrh) [inline cli-paho-java/src/main/java/com/redhat/mqe/amc/Client.java]: I'd remove the empty line here

File: cli-paho-java/src/main/java/com/redhat/mqe/amc/Client.java [hunk 3]
```diff
@@ -175,24 +187,35 @@ void printMessage(String topic, MqttMessage message) {
...
+                client.close();
+                throw e;
+            }
         }
     }
 
     protected Logger setUpLogger(String name) {
         Logger log = Logger.getLogger(name);
-        log.setLevel(Level.INFO);
+        log.setLevel(Level.WARN);
```
Comment (jdanekrh) [inline cli-paho-java/src/main/java/com/redhat/mqe/amc/Client.java]: We will need an option to have logging enabled from commandline. Can you create a Jira for yourself (or github issue, whatever, just so we dont forget) and assign it to yourself?

**Comment by jdanekrh** [comment]
We can split the commit tomorrow. @rvais How would you do it? My strategy would be `git reset --soft HEAD^` then commit the first using `git gui` (commit individual lines), then commit second, then force push into the origin branch.

**Comment by rvais** [comment]
I would have to try it, because I don't know what exact effect it will have on commit IDs and whether or not you will need to do a pushforce to override the changes that are in the remote branch. But in general it seems like good solution. (Definitely better than what I did few times :D )

File: cli-paho-java/src/main/java/com/redhat/mqe/amc/Client.java (hunk 3 above)
Comment (jdanekrh) [reply cli-paho-java/src/main/java/com/redhat/mqe/amc/Client.java]: MSGQE-6001 Create option for enabling logging from command line in cli-paho-java

**Pull request merged**